src/main/java/com/github/manolo8/darkbot/backpage/DispatchManager.java

```diff
@@ -82,19 +84,66 @@ public boolean hireRetriever(Retriever retriever) {
     }
 
     public boolean collectInstant(InProgress progress) {
+        if (progress == null) return false;
         try {
             System.out.println("Collecting Instant: Slot " + progress.getSlotId());
             if (data.getPrimeCoupons() <= 0)
-                return handleResponse("Cannot instant collect", progress.getId(),
-                        "(ERROR) No Prime Coupon available for instant collection");
+                return handleResponse("Instant Collect", progress.getId(),
+                        "(ERROR) Can Not Instant Collect, No Prime Coupon available for instant collection");
             String response = main.backpage.getConnection("ajax/dispatch.php", Method.POST)
                     .setRawParam("command", "instantComplete")
                     .setRawParam("dispatchId", progress.getId())
                     .setRawParam("dispatchRewardPackage", progress.getDispatchRewardPackage())
                     .setRawParam("slotId", progress.getSlotId())
                     .getContent();
 
-            return handleResponse("Collected retriever", progress.getId(), response);
+            return handleResponse("Collected Retriever", progress.getId(), response);
+        } catch (Exception e) {
+            e.printStackTrace();
+            System.out.println("Exception collecting dispatcher: " + e);
```

Comment:
i think it's better when these are the other way arround, so you first have the message then the following stacktrace